tests/installation/ipxe_install.pm

```diff
@@ -70,13 +70,23 @@ sub set_bootscript {
     my $arch = get_required_var('ARCH');
     my $autoyast = get_var('AUTOYAST', '');
     my $mirror_http = get_required_var('MIRROR_HTTP');
+    my $openqa_hostname = get_var('OPENQA_HOSTNAME', 'openqa.suse.de');
 
     # trim all strings from variables to get rid of bogus whitespaces
     $arch =~ s/^\s+|\s+$//g;
     $autoyast =~ s/^\s+|\s+$//g;
     $mirror_http =~ s/^\s+|\s+$//g;
+    $openqa_hostname =~ s/^\s+|\s+$//g;
 
+    # For pxe/ipxe agama installation,we don't have linuxrc 'install=' parameter
+    # so we need to pass iso image to boot
     my $install = "install=$mirror_http";
+    if (is_agama) {
+        my $agama_iso = get_required_var('ISO');
+        $agama_iso =~ s/^\s+|\s+$//g;
+        my $agama_live_iso = get_var("AGAMA_LIVE_ISO", "http://$openqa_hostname/assets/iso/$agama_iso");
```

Comment:
I think we could use here `REPO_999` instead of `AGAMA_LIVE_ISO` it is not a great name but it is how it is extracted from obs sync, then we could remove another `AGAMA_*` setting.

Reply:
So can you please help add comments at https://github.com/os-autoinst/openqa-trigger-from-obs/pull/281 before I change the code here?

I think we can keep all platforms there `REPO_999`, but I am not sure

Reply:
we actually asked for that to not be blocked, because the url was too long in s390x zvm for the s390x console, because the iso is under isos and if you can use it from there should be enough, this is like secondary location for this special case.

Reply:
Ahm ok, you cannot use, you need the iso under some server now I remember.